code/game/gamemodes/clock_cult/clock_effects/servant_blocker.dm

```diff
@@ -13,8 +13,9 @@
 
 /obj/effect/clockwork/servant_blocker/CanPass(atom/movable/M, turf/target)
 	var/list/target_contents = M.GetAllContents() + M
+	var/obj/structure/destructible/clockwork/massive/celestial_gateway/Ark = GLOB.ark_of_the_clockwork_justiciar
 	for(var/mob/living/L in target_contents)
-		if(is_servant_of_ratvar(L) && get_dir(M, src) != dir) //Unless we're on the side the arrow is pointing directly away from, no-go
+		if(is_servant_of_ratvar(L) && get_dir(M, src) != dir && L.stat != DEAD && Ark && Ark.progress_in_seconds < GATEWAY_REEBE_FOUND) //Unless we're on the side the arrow is pointing directly away from, no-go
```

Comment:
Why are you checking the time of the ark? For that matter, why does this let servants go in after 4 minutes, and why is that not mentioned?

Reply:
Correct me if I'm wrong but I thought that DEFINE was the point where the gateways to Reebe start showing up. I wanted to prevent any shenanigans before the crew showed up but it seems like this check is more trouble than its worth.